### ISSUE TITLE
Hi, In train_test.py，Is the valid dataset also used to train the model?

### ISSUE THREAD

**Issue by bitwangdan** [ASMIftekhar/VSGNet#14]
![image](https://user-images.githubusercontent.com/18326564/101017142-a9866d00-35a4-11eb-8e5f-abf60a28a7cd.png)

**Comment by ASMIftekhar** [comment]
Yes, as mentioned in the paper, like previous works we train in both training and validation set in the V-COCO dataset.
